megabus/src/main/java/com/bazaarvoice/megabus/MegabusRef.java

```diff
@@ -26,14 +26,17 @@
     private final String _key;
```

Comment:
also add json serialization/deserialization tests to ensure that changes are backward compatible

Reply:
will do.